main_logic/omni_offline_client.py

```
@@ -271,58 +292,84 @@ async def stream_text(self, text: str) -> None:
+                                discard_reason or "guard",
+                                guard_attempt,
+                                self.max_response_rerolls,
+                                will_retry,
+                                failure_message
+                            )
+                            
+                            if will_retry:
+                                logger.info(f"OmniOfflineClient: 响应被丢弃（{discard_reason}），第 {guard_attempt}/{self.max_response_rerolls} 次重试")
+                                continue
+                            
+                            logger.warning("OmniOfflineClient: guard 重试耗尽，放弃输出")
```

Ruff (0.15.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)